RELEASE_NOTES.md

```diff
@@ -1,3 +1,95 @@
+## Cookstyle 5.10
+
+### 9 New Chef Cops
+
+#### ChefDeprecations/VerifyPropertyUsesFileExpansion
+
+The `VerifyPropertyUsesFileExpansion` cop detects resources using the verify property to validate file content with the legacy `file` variable instead of the `path` variable required in Chef Infra Client 13 and later.
```

Comment:
We're trying to format property names also, correct?
```suggestion
The `VerifyPropertyUsesFileExpansion` cop detects resources using the `verify` property to validate file content with the legacy `file` variable instead of the `path` variable required in Chef Infra Client 13 and later.
```